bucket/fsresizer.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "4.3",
+    "description": "An image converter and renaming tool",
+    "homepage": "https://www.faststone.org/FSResizerDetail.htm",
+    "license": {
+        "identifier": "Shareware",
+        "url": "https://www.faststone.org/order.htm"
+    },
+    "url": "http://www.faststonesoft.net/DN/FSResizer43.zip",
+    "hash": "4ec749cb49ee27ede8dea664e3b1c868b64875d94e2dda3a7aa4c610d5bd5c63",
+    "extract_dir": "FSResizer43",
+    "pre_install": "if (!(Test-Path \"$persist_dir\\FSResizerSettings.db\")) { New-Item \"$dir\\FSResizerSettings.db\" | Out-Null }",
+    "bin": "FSResizer.exe",
+    "shortcuts": [
+        [
+            "FSResizer.exe",
+            "FSResizer"
```

Comment:
```suggestion
            "FastStone Photo Resizer"
```

![image](https://user-images.githubusercontent.com/13260377/76208591-45df3700-6200-11ea-918a-b6631369ffec.png)

Reply:
This name pattern follows [FSCapture](https://github.com/lukesampson/scoop-extras/blob/master/bucket/fscapture.json) and [FSViewer](https://github.com/lukesampson/scoop-extras/blob/master/bucket/fsviewer.json).

- FSCapture ==> FastStone Capture
- FSViewer ==> FastStone Image Viewer